### ISSUE TITLE
variable accesses are not saved

### ISSUE THREAD

**Issue by ret2libc** [radare/radare2#4619]
When we dump the variables/arguments of a function, we just use `afv`/`afa`, without dumping the info about references.

I think in `r_anal_var_list_show` we should also print `afag`/`afvg` commands, when in '*' mode.
This would require that in `r_anal_var_list`, when we create each `RAnalVar` from SDB, we use also the `accesses` field of RanalVar structure.

**Comment by oddcoder** [comment]
not sure what do you mean by var access please add extra info about what are you trying to achieve

**Comment by radare** [comment]
As long as all this info is saved now in sdb we should do the same as done for xrefs. This is, saving that sdb in the project directory and loased back after this

**Comment by ret2libc** [comment]
@oddcoder @radare that's right. I would like to have variables information saved in the project files.
You can see the problem in this way:
```
$ r2 ./radare2-regressions/bins/elf/ls-linux64
[0x00005c20]> s main
[0x00003ad0]> af
[0x00003ad0]> k anal/fcns/*~vars
inst.0x4da0.vars=0x3ad0,a,0x1,0x12
inst.0x4bab.vars=0x3ad0,a,0x1,0x2e
inst.0x4bf6.vars=0x3ad0,a,0x1,0x2e
inst.0x4bfd.vars=0x3ad0,a,0x1,0x0
inst.0x5a13.vars=0x3ad0,a,0x1,0x0
[0x00003ad0]> Ps myvartest
myvartest
[0x00003ad0]> q
Do you want to save the 'myvartest' project? (Y/n)

$ r2 -p myvartest ./radare2-regressions/bins/elf/ls-linux64
[0x00003ad0]> k anal/fcns/*~vars
[0x00003ad0]> 
```

This shows how the inst.XXXX.vars information are lost while saving/loading the project. Those data are used in particular when you try to rename variables in visual mode. They should be saved in the project.
If you look in the project file, you will see that we just use the `afa`/`afv` commands, but not the `afag`/`afas`/`afvg`/`afvs` commands, so the references are not saved.

IMHO it's enough to make `afa*`/`afv*` commands print also the `afag`|`afas`/`afvg`|`afvs` commands.

**Comment by radare** [comment]
ping @oddcoder

**Comment by radare** [comment]
the accesses are saved already. they are just not lisetd, this is very easy to fix.

the same goes for .writes=.. so its just a matter of walk enumerate the variables of the function and then resolving the key "var.${fcnaddr}.${type}.${index}.reads"  and list the offsets in the array.

The output of afvs and afvg should be some thing like:

```
var int local_34h @ rbp-0x34
   read 0x80480 0x804801 0x9044802
   write 0x80474
var int local_30h @ rbp-0x30
  ...
```
fmi:
```
$ r2 /bin/ls
[0x100001174]> af
[0x100001174]> k anal/fcns/*~reads
var.0x100001174.1.-1088.reads=0x100001730
var.0x100001174.1.-1600.reads=0x10000118e
var.0x100001174.1.-1608.reads=0x10000174f,0x100001769,0x100001783,0x10000179d,0x1000017b7,0x1000017d6
var.0x100001174.1.-1620.reads=0x10000187c
var.0x100001174.1.-1624.reads=0x10000191f
var.0x100001174.1.-1632.reads=0x100001a07,0x1000019c5,0x100001993
var.0x100001174.1.-46.reads=0x10000120c
var.0x100001174.1.-48.reads=0x1000011f2
var.0x100001174.1.-52.reads=0x1000018e6
```

**Comment by radare** [comment]
afvs is already handled and have a different meaning. so i used afvR and afvW


Fixed in 37e79b3ba1501738550624f660cf09d976a56d69
